docs/python/tutorials/classes.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 15
+title: Decorating Class Methods
+---
+
+You can add DBOS workflow, step, and transaction decorators to your Python class methods.
```

Comment:
Again, the use of "class methods" is ambiguous here... We can say "You can use DBOS decorators to functions defined in Python classes".